src/target/riscv/riscv-013.c

```diff
@@ -1017,6 +1046,32 @@ static int read_abstract_arg(struct target *target, riscv_reg_t *value,
 	return result;
 }
 
+/**
+ * Queue scans into a batch that write the value to abstract data registers:
+ * data[index] (and data[index+1] in case of 64-bit value).
+ *
+ * No extra DTM delay is added after the write to data[N]. It is assumed that
+ * this is a one-shot abstract command, that means no auto-execution is set up
+ * (abstractauto.autoexecdata bits are zero).
+ */
+static void abstract_data_write_fill_batch(struct riscv_batch *batch,
```

Comment:
To align it with the proposed names above, this one could be named:

```suggestion
static void abstract_cmd_batch_fill_data_write(struct riscv_batch *batch,
```

Reply:
I'd like to reuse it in `write_abstract_arg`. I think it will be quite confusing if it were named `abstract_cmd_batch_fill_data_write`

Reply:
Ok, let's leave the name as is, then.